hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -54,6 +54,14 @@ public class HoodieCommonConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Enables support for Schema Evolution feature");
 
+  public static final ConfigProperty<Boolean> SCHEMA_EVOLUTION_ALLOW_LOGICAL_EVOLUTION = ConfigProperty
+      .key("hoodie.schema.evolution.allow.logical.evolution")
```

Comment:
why need this flag? timestamp-millis to/from timestamp-micros should always be feasible in schema evolution.

Reply:
Then why we need this flag if the precision loss is never allowed in schema evolution. Just abadon the converion in the code?

BTW, not only timestamp type got logical types in avro schema right? and I didn't see it works for preventing precision loss from this name. At least type promotion should be allowed.